puppet/modules/jenkins_job_builder/manifests/config.pp

```diff
@@ -6,6 +6,11 @@
   String $password,
   Integer[0] $jenkins_jobs_update_timeout = 600,
   String $command_arguments = 'foreman-infra-jenkins-job-update',
+  String $git_project_name = 'foreman-infra',
+  String $git_repo = 'https://github.com/theforeman/foreman-infra.git',
+  String $git_branch = 'master',
```

Comment:
It appears that this isn't used anywhere @ehelms . What is the intention?

Reply:
Dropped the parameter